src/components/control-bar/control-bar-items/MeltMenuItem.svelte

```diff
@@ -0,0 +1,27 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+
+  SPDX-License-Identifier: MIT
+ -->
+
+<style lang="postcss">
+  .item[data-highlighted] {
+    @apply bg-gray-100;
+  }
+</style>
```

Comment:
Could not find a way of inlining this kind of data attribute styling. Also, tried doing it as such:
```
  .item {
    @apply data-[highlighted]:bg-gray-100;
  }
```
as demonstrated in [melt ui dropdown example code](https://melt-ui.com/docs/builders/dropdown-menu), but it was not working and got an "Semicolon or block is expected" error

Reply:
I think this is fine. WindiCSS does not support data-[highlighted]:bg-gray-100 - not a valid class.